src/trusted/service_runtime/sel_ldr.c

```diff
@@ -142,13 +142,13 @@ int NaClAppWithSyscallTableCtor(struct NaClApp               *nap,
   nap->initial_entry_pt = 0;
   nap->user_entry_pt = 0;
 
-  if (!DynArrayCtor(&nap->threads, 2)) {
+  if (!DynArrayCtor(&nap->threads, 16)) {
     goto cleanup_cpu_features;
   }
-  if (!DynArrayCtor(&nap->desc_tbl, 2)) {
+  if (!DynArrayCtor(&nap->desc_tbl, 16)) {
     goto cleanup_threads;
   }
-  if (!DynArrayCtor(&nap->children, 2)) {
+  if (!DynArrayCtor(&nap->children, 16)) {
```

Comment:
This is just the default starting capacity of the dynamic array. Was there any reason you did this?